### PR TITLE
Docs/react examples

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

This PR includes React examples as well as cosmetic fixes (see the respective commits separately).

## Type of change

Please delete options that are not relevant.

- [x] Bugfix
- [x] Documentation content changes

## Is it a breaking change?

- [x] No

# How Has This Been Tested?

Please describe the tests that you've added and run to verify your changes. 
Provide instructions, so we can reproduce.

- [x] e2e tests
- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-checkbox/readme.md [hunk 1]
```diff
@@ -182,11 +196,13 @@ If the `indeterminate` attribute is present on the `ld-checkbox` component, the
 {% example '{ "background": "light" }' %}
 <ld-checkbox indeterminate></ld-checkbox>
 
-<ld-checkbox indeterminate disabled></ld-checkbox>
```
Comment (renet) [inline src/liquid/components/ld-checkbox/readme.md:185]: Seems like you removed the `indeterminate disabled` component from the example. Is that intended?

File: src/liquid/components/ld-checkbox/readme.md [hunk 2]
```diff
@@ -242,7 +238,9 @@ document.querySelectorAll('#example-indeterminate-1 input, #example-indeterminat
 {% example '{ "background": "light" }' %}
 <ld-checkbox tone="dark"></ld-checkbox>
 
-<ld-checkbox tone="dark" disabled></ld-checkbox>
```
Comment (renet) [inline src/liquid/components/ld-checkbox/readme.md:245]: Is removing the disabled component from the example intended?

File: src/liquid/components/ld-checkbox/readme.md [hunk 3]
```diff
@@ -394,7 +336,9 @@ The checkbox in mode "danger" looks and behaves the same as a checkbox with the
 {% example %}
 <ld-checkbox mode="danger"></ld-checkbox>
 
-<ld-checkbox mode="danger" disabled></ld-checkbox>
```
Comment (renet) [inline src/liquid/components/ld-checkbox/readme.md:397]: Same question as before.

File: src/liquid/components/ld-circular-progress/readme.md [hunk 4]
```diff
@@ -74,6 +82,37 @@ Interactive example:
...
+        onLdchange={(ev) => {
+          setVal(ev.detail[0])
+        }}
+        value={val}
+        max={200}
+        width="14rem"
+      />
+    </>
+  )
+}
```
Comment (renet) [inline src/liquid/components/ld-circular-progress/readme.md:114]:
```suggestion
const [val, setVal] = useState(25)
return (
  <>
    <LdSrOnly id="progress-label">Progress</LdSrOnly>
    <LdCircularProgress aria-labelledby="progress-label" aria-valuenow={val}>
      <LdTypo
        variant="b6"
        style={ {
          color: `var(--ld-thm-${val > 100 ? 'error' : 'primary'})`,
        } }
      >
        {val}%
      </LdTypo>
      <LdTypo variant="label-s">complete</LdTypo>
    </LdCircularProgress>

    <LdSlider
      onLdchange={(ev) => {
        setVal(ev.detail[0])
      }}
      value={val}
      max={200}
      width="14rem"
    />
  </>
)
```

File: src/liquid/components/ld-circular-progress/readme.md [hunk 5]
```diff
@@ -273,6 +358,91 @@ You can change the size of the circular progress bar. However, with a smaller si
...
+    place-items: center;
+    gap: var(--ld-sp-4);
+  }
+  .report ld-circular-progress {
+    --ld-circular-progress-size: 4rem;
+  }
+  .report-value {
+    transform: scale(0.8);
+  }
+`}</style>
```
Comment (renet) [inline src/liquid/components/ld-circular-progress/readme.md:375]: I did it the same way first, but wasn't really happy with that. Nobody really does it that way with React, I guess. I ended up with assigning a styles object to a variable and assigning that variable to the `style` prop of the components. Maybe you wanna do it the same way, here?

```jsx
const reportStyles = {
  display: 'grid'
  placeItems: 'center'
  gap: 'var(--ld-sp-4)'
}

// and so on ...

return (
  <>
    <div style={reportStyles}>
      {/* ... */}
```

File: src/liquid/components/ld-cookie-consent/readme.md [hunk 6]
```diff
@@ -102,6 +102,94 @@ The `ld-cookie-consent` Web Component is configurable using the `settings` prop.
 <ld-button onclick="event.target.previousElementSibling.showDisclaimer()">
   Show cookie consent disclaimer
 </ld-button>
+
+<!-- React component -->
+
+const App = () => {
```
Comment (renet) [inline src/liquid/components/ld-cookie-consent/readme.md:108]: Same as above, for all the examples in this file.
```suggestion
```

File: src/liquid/components/ld-cookie-consent/readme.md [hunk 7]
```diff
@@ -650,6 +1176,129 @@ You can theme the component using CSS custom properties and customize its conten
...
+    privacyStatementURL: 'legal/privacy/',
+    showOnLoad: false,
+  }
+
+  return (
+    <>
+      <LdCookieConsent
+        ref={cookieConsentRef}
+        settings={cookieConsentSettings}
+        style={ {
```
Comment (renet) [inline src/liquid/components/ld-cookie-consent/readme.md:1259]: maybe move this styles object to a variable, too?

File: src/liquid/components/ld-cookie-consent/readme.md [hunk 8]
```diff
@@ -650,6 +1176,129 @@ You can theme the component using CSS custom properties and customize its conten
...
+        <svg {...{slot: 'preferences-logo'}} width="901" height="870" viewBox="0 0 901 870" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M461 174.8c133.6 0 244.8 97.4 263.5 224.8h176C881.9 176.1 692 0 461 0v174.8Zm221-70c-1.2 0-1.2 0 0 0Z" fill="#5F82FF"/><path d="M461 694.8c133.6 0 244.8-97.4 263.5-224.8h176C881.9 693.5 692 869.6 461 869.6V694.8Zm221 70c-1.2 0-1.2 0 0 0Z" fill="#EA788E"/><path d="M0 434.7c-.4 110 41 216 115.8 296.8A436 436 0 0 0 403 870V697.1A265 265 0 0 1 173 435a264.1 264.1 0 0 1 230-262.1V0a436 436 0 0 0-287 138.3A434.7 434.7 0 0 0 0 434.7Z" fill="#7959B8"/></svg>
+      </LdCookieConsent>
+
+      <LdButton onClick={(ev) => cookieConsentRef?.current?.showDisclaimer()}>
```
Comment (renet) [inline src/liquid/components/ld-cookie-consent/readme.md:1296]: Please remove TS conditionals.

File: src/liquid/components/ld-cookie-consent/readme.md [hunk 9]
```diff
@@ -529,6 +1005,31 @@ You can align the cookie consent disclaimer to the left or right of the screen.
...
+  }
+
+  return (
+    <>
+      <LdCookieConsent
+        ref={cookieConsentRef}
+        settings={cookieConsentSettings}
+      />
+
+      <LdButton onClick={(ev) => cookieConsentRef?.current?.showDisclaimer()}>
```
Comment (renet) [inline src/liquid/components/ld-cookie-consent/readme.md:1027]: Please remove TS conditionals.

File: src/liquid/components/ld-cookie-consent/readme.md [hunk 10]
```diff
@@ -544,6 +1045,31 @@ You can align the cookie consent disclaimer to the left or right of the screen.
...
```
Comment (renet) [inline src/liquid/components/ld-cookie-consent/readme.md:1067]: Please remove TS conditionals.

File: src/liquid/components/ld-slider/ld-slider.tsx [hunk 11]
```diff
@@ -94,7 +94,9 @@ export class LdSlider implements InnerFocusable {
   /** Adds custom stop points to the slider (instead of steps) */
   @Prop() unit?: string
   /** Specifies the default value */
-  @Prop({ mutable: true, reflect: true }) value?: string = String(this.min)
+  @Prop({ mutable: true, reflect: true }) value?: string | number = String(
+    this.min
+  )
```
Comment (renet) [inline src/liquid/components/ld-slider/ld-slider.tsx:99]:
```suggestion
  @Prop({ mutable: true, reflect: true }) value?: string | number = this.min
```

Wouldn't this work as well, then?

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-checkbox/readme.md (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/components/ld-checkbox/readme.md:185]: Yes, this is intended. Disabled is explained above and I realized there is no need to display each following example with a disabled state. It also is more user friendly for copy pasting examples and looks less cluttered.

File: src/liquid/components/ld-checkbox/readme.md (hunk 2 above)
Comment (borisdiakur) [reply src/liquid/components/ld-checkbox/readme.md:245]: Yes, see my comment above ☝️

File: src/liquid/components/ld-checkbox/readme.md (hunk 3 above)
Comment (borisdiakur) [reply src/liquid/components/ld-checkbox/readme.md:397]: Yes, see my comment above ☝️

**Review by renet — APPROVED** [review]

**Pull request merged**